### ISSUE TITLE
Muted player unmutes based on cross-site setting

### ISSUE THREAD

**Issue by CrazyLittle** [FabricLabs/soundtrack#105]
$.cookie('lastVolume') = “0”

It appears that the youtube player volume leaks outside of SIO, so if you mute SIO and then go to youtube, it mutes youtube, and vice versa - unmute youtube and SIO unmutes on next play.

To reproduce:  
1) Mute SIO
2) Go to Youtube, play a video (it will be muted as well), un-mute the youtube video.
3) Wait for SIO to play another youtube song, SIO will be un-muted even though $.cookie('lastVolume') = “0” and the volume bar shows mute.

**Comment by zbeekman** [comment]
@naterchrdsn Are you sure you weren’t experiencing #106?

**Comment by naterchrdsn** [comment]
yes, I had both youtube and s.io open at one point and noticed this bug
